### ISSUE TITLE
Consumed and Produced values for phases seem reversed?

### ISSUE THREAD

**Issue by arjan127** [reneklootwijk/node-dsmr#5]
First off, thanks for this package. It's a real lifesaver!

While using it, I however noticed the following: 

Take the following values:
```
    instantaneousProducedElectricityL1: 0,
    instantaneousProducedElectricityL2: 0.476,
    instantaneousProducedElectricityL3: 0.571,
    instantaneousConsumedElectricityL1: 0.334,
    instantaneousConsumedElectricityL2: 0,
    instantaneousConsumedElectricityL3: 0
```
And looking at the `actualConsumed` and `actualProduced` values:
```
    actualConsumed: 1.047,
    actualProduced: 0.334,
```

It appears that the produced phase values add up to the `actualConsumed` value, and likewise the consumed values add up to the `actualProduced` value. This is consistently the case, without fail. 

Perhaps this is intended (in which case could you explain why?), but I wanted to bring it to light in case it's a simple mistake. Thanks!

**Comment by reneklootwijk** [comment]
Thanks for your observation. I misinterpret the P+ and P- in the specification. Since yesterday I have a 3 phase connection and could actually see it. It has been fixed but not released yet, because I am testing some other changes (validating the checksum) before committing.

**Comment by reneklootwijk** [comment]
Fixed in version 0.1.18
